sdk/src/jumbf_io.rs

```diff
@@ -12,64 +12,79 @@
 // each license.
 
 use std::{
+    collections::HashMap,
     fs::{self, File},
     io::Cursor,
     path::{Path, PathBuf},
 };
 
+use lazy_static::lazy_static;
```

Comment:
Need to pause review here. Will continue later today or tomorrow as time permits.